index.html

```diff
@@ -535,11 +537,15 @@
 Developing the necessary understanding of different uses of new technology &mdash;
 and then evolving norms to handle those situations &mdash;
 is not something that can be rushed.
-[Deployment](https://developer.apple.com/videos/play/wwdc2025/232/) [is](https://developer.chrome.com/blog/digital-credentials-api-origin-trial).
 
 
 ### Authorizing Sites {#auth}
 
+Websites must only request digital credentials
+when that level of proof provides user benefit.
```

Comment:
```suggestion
when absolutely warranted or required by law.
```

Reply:
Not being party to violating the law can be a user benefit.  See also KYC.  I think we're good here.